docs/k0s-install.md

```diff
@@ -0,0 +1,306 @@
+# Install k0s as a service
+
+There are two ways to run k0s binary as a service
+
+1. Using `k0s install` and
+2. Using `k0sctl` tool
+
+## k0s install
+
+`install` command is a helper command for setting up k0s on a brand-new system. It will detect install system init utility installed on the desired host. Supported system inits are systemd and open-rc. NOTE: Windows is currently not supported
+
+Install can be executed for two k0s roles: controller and worker.
+
+Install `controller` subcommand will accept all available flags and parameters supported.
```

Comment:
> will accept all available flags and parameters supported.

and will not accept unavailable flags and unsupported parameters? :)

I guess this needs some clarification, like "will accept the same flags as `k0s server`" 

Does it need to be mentioned they're the same, I think they're shown in the help for `k0s install controller`?